### DISCUSSION TITLE
Direct import of .ics files into Google Calendar with Provider

### DISCUSSION THREAD

**Discussion by dellow3** [kewisch/gdata-provider#499] [Support] [answered]
I'd like to be able to open .ics event files that are emailed to me so they are entered directly into Google Calendar via Provider or some other mechanism. Is this possible?  How?

**Comment by kewisch** [comment, marked as answer]
You should be able to import a calendar from a saved file using Thunderbird, it doesn't matter which type of calendar it goes to. In the Events & Tasks menu select "Import"
